Sources/SwiftParser/Expressions.swift

```diff
@@ -1853,7 +1853,7 @@ extension Parser {
   ///
   /// This is currently the same as parsing a tuple expression. In the future,
   /// this will be a dedicated argument list type.
-  mutating func parseArgumentListElements(pattern: PatternContext, flavor: ExprFlavor = .basic) -> [RawLabeledExprSyntax] {
+  mutating func parseArgumentListElements(pattern: PatternContext, flavor: ExprFlavor = .basic, allowTrailingComma: Bool = true) -> [RawLabeledExprSyntax] {
```

Comment:
I’m generally not a fan of defaulted arguments for these kinds of options because it’s too easy to just forget to pass it down from a different call or not think about it. I’d prefer to be explicit here and require it in the calls.